test/e2e/minscale_readiness_test.go

```diff
@@ -67,7 +66,15 @@ func TestMinScale(t *testing.T) {
 		t.Fatalf("The deployment %q did not scale >= %d before becoming ready: %v", deploymentName, minScale, err)
 	}
 
+	// Before becoming ready, observe revision endpoints
+	endpointName := revName + "-private"
```

Comment:
Can we fetch this via API? See https://github.com/knative/serving/blob/afbd55a1df2d1548f603bd249b4672141beb8007/test/e2e/autoscale_test.go#L283-L302 for an example of how to do this.